aiven/client/argx.py

```diff
@@ -186,7 +250,7 @@ def expected_errors(self):
     def print_response(
         self,
         result,
-        json=True,
+        json=False,
```

Comment:
The basic problem is this default is kind of silly, because AFAIK everywhere else this is used `json` is overridden by `args.json` which is a command line flag that evals to `False`. So conventionally, anywhere in our code we a have an explicit `--json` flag on the command, this is actually defaulting to false.

From a UI consistency perspective this isn't great: if you add a new command and use `print_response` naively, unless you stick an argx.json flag on there, you get JSON by default rather then pretty table everywhere there _is_ a `--json` option.

I suppose to backwards compatibility this, `json=None` could be specified, and we detect that as `json=True` (but unspecified) which means the code can detect if a default was specified and then choose whether to use the global format selection if _that_ was specified.

Reply:
Agree it is a silly default. I guess the main thing is we don't change the output accidentally and break any user code/scripts. If you can be sure that this change won't have that effect then it should be OK.

Reply:
Need a call on this one then. A non-thorough (i.e. programmatic) look through our existing interface suggests it would change nothing about the commands in `aiven-client`.

Reply:
@ojarva do you have an opinion here?